services/gateway/namespaces.js

```diff
@@ -36,6 +36,7 @@ module.exports = filterApis(config.api.ws, {
 	'/rpc': () => registerApi(['http-version1', 'http-version1-compat'], { ...defaultConfig }),
 	'/rpc-v1': () => registerApi(['http-version1', 'http-version1-compat'], { ...defaultConfig }),
 	'/rpc-v2': () => registerApi(['http-version2'], { ...defaultConfig }),
+	'/rpc-status': () => registerApi(['http-status'], { ...defaultConfig }),
```

Comment:
The RPC does not need status at all, it's only for HTTP.
WS-RPC preserves state of the client and some basic equivalent of this is hardcoded in the WebSocket implementation.